02-express-tutorial/answers/02-http-app.js

```diff
@@ -0,0 +1,59 @@
+const http = require('http')
+const { readFileSync } = require('fs')
+
+// get all files
+const homePage = readFileSync('./navbar-app/index.html')
+const homeStyles = readFileSync('./navbar-app/styles.css')
+const homeImage = readFileSync('./navbar-app/logo.svg')
+const homeLogic = readFileSync('./navbar-app/browser-app.js')
```

Comment:
What would happen as the number of pages continues to grow? How would this implementation affect the behavior of our server?